easybuild/easyconfigs/f/fastQValidator/fastQValidator-0.1.1a-dev-goolf-1.7.20-aadc6f9.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'fastQValidator'
+version = '0.1.1a-dev'
```

Comment:
I got this from their makefile and added the -dev
https://github.com/statgen/fastQValidator/blob/master/Makefile.inc#L5

I could use a timestamp if you prefer